src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoEventBox.cs

```diff
@@ -7,6 +7,23 @@ namespace Uno.UI.Runtime.Skia.Gtk;
 
 internal class UnoEventBox : EventBox
 {
+	public UnoEventBox()
+	{
+		var swipe = new GestureSwipe(this);
+		swipe.End += Swipe_End;
+		swipe.Begin += Swipe_Begin;
```

Comment:
Those kind of handling has proven to be difficult to maintain / make uniform on all targets, we should avoid them. Instead we should listen to raw pointer events and convert then interpret the gestures like `Swipe` (cf. `GestureRecognizer`)

Reply:
Not currently relying on them. Will cleanup